packages/flutter/lib/src/widgets/heroes.dart

```diff
@@ -425,13 +426,15 @@ class _HeroFlightManifest {
   final CreateRectTween createRectTween;
   final HeroFlightShuttleBuilder shuttleBuilder;
   final bool isUserGestureTransition;
+  final bool isDiverted;
```

Comment:
Is this documented somewhere else?  It's not immediately obvious what it means.

Reply:
It's documented more prominently here: https://github.com/flutter/flutter/blob/556e3d9e7486dae8f158bc432e82c1267e873846/packages/flutter/lib/src/widgets/heroes.dart#L588-L590